### ISSUE TITLE
I can't see apple pay option 

### ISSUE THREAD

**Issue by codal-hkrishnani** [wgltony/react-native-braintree-dropin-ui#83]
I tried all steps given in [here](https://developer.paypal.com/braintree/docs/guides/apple-pay/configuration/ios/v5) but still not able to see apple pay option. Any help will be appreciated.

**Comment by Vimal963** [comment]
Any update?

**Comment by codal-hkrishnani** [comment]
Please refer [this](https://developer.apple.com/apple-pay/sandbox-testing/) link might help.. Or let me know what exact error are you facing.

**Comment by Vimal963** [comment]
Yes, On the debug mode apple pay option is visible but whenever I upload the application on testflight and run on release mode then apple pay option is not visible on the payment.

**Comment by codal-hkrishnani** [comment]
@Vimal963 User must have cards added in iphone wallet. Can you please check you have cards added or not?

**Comment by Vimal963** [comment]
Now apple pay and paypal is show but whenever we click on the paypal and braintree then nothing happen 

Could you please help me

**Comment by Aksh0377** [comment]
I have followed all the steps but still not able to see  apple pay option, can you please help ?

FYI,

1. I have setup braintree as US merchant
2. Configured apple certificates
3. set device region as UK
4. added test visa card details in apple wallet

still it is not working.
